components/review/stages/reviewInfoStage.tsx

```diff
@@ -1,12 +1,162 @@
+  return (
+    <div className="py-[12px]">
+      <p className="text-[16px] font-poppins pb-[4px]">{question}</p>
+      <p className="text-[16px] font-source text-charcoal-500">{answer}</p>
+    </div>
+  );
+};
+
+type ConflictModalProps = {
+  readonly name: string;
+  readonly open: boolean;
+  readonly onClose: () => void;
+};
+
+/** Highlighted project modal */
+const ConflictModal: FC<ConflictModalProps> = ({ name, open, onClose }) => {
+  return open ? (
+    <>
+      {/* Overlay */}
+      <div
+        className="fixed inset-0 flex justify-center items-end md:items-center md:p-4 bg-black bg-opacity-20 z-40"
+        style={{ margin: "0" }}
+        onClick={onClose}
+      >
+        {/* Modal */}
+        <div
+          className="md:max-w-[310px] relative flex flex-col px-6 md:px-12 pt-4 pb-4 rounded-t-3xl md:rounded-3xl bg-white shadow-lg"
+          onClick={(e) => e.stopPropagation()}
+        >
+          {/* Modal content */}
+          <div className="flex justify-between space-x-8 ">
+            {/* Description */}
+            <div className="flex flex-col justify-between space-y-4 text-center">
+              <div>
+                <h6 className="mb-1 text-blue">Report a conflict</h6>
+                <p className="text-[16px] font-source text-charcoal-500">
+                  Do you know {name} and want to report a conflict?
+                </p>
+              </div>
+
+              <div className="flex justify-center space-x-6">
+                <Button
+                  variant="secondary"
+                  onClick={onClose}
+                  className="whitespace-nowrap"
+                >
+                  No
+                </Button>
+                <Button
+                  variant="primary"
+                  onClick={onClose}
+                  className="whitespace-nowrap"
+                >
+                  Yes
+                </Button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </>
+  ) : null;
+};
 
 export const ReviewInfoStage: React.FC = () => {
+  const [modalOpen, setModalOpen] = useState(false);
+
   return (
     <ReviewSplitPanelPage
-      studentName="M. Goose"
+      studentName="Matthew Wang"
       rightTitle="Basic Information"
+      rightTitleButton={
+        <Button
+          variant="secondary"
+          onClick={() => setModalOpen(true)}
+          className="whitespace-nowrap"
+          size="sm"
+        >
+          <div className="flex place-items-center space-x-2">
+            <WarningIcon />
+            <p>Conflict</p>
+          </div>
+        </Button>
+      }
       currentStage={ReviewStage.INFO}
+      leftContent={
+        <div className="flex flex-col place-items-center space-y-8 place-content-center h-full">
+          <div>
+            <Image
+              height={87}
+              width={440}
+              alt="Picture of the Blueprint logo"
```

Comment:
Since this is a decorative image, we should use `alt=""` to improve the experience for anyone who needs alt tags.  By setting it intentionally blank, the screen reader knows to skip the image content.